src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,160 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
```

Comment:
Private fields should be after public ones in your class. Move the `Node<T>` class definition to the end of the `MyLinkedList` class.

Reply:
I don't have public fields in code